client/files.go

```diff
@@ -110,3 +115,56 @@ func WriteUserSignedKey(signedKey string, key string) (err error) {
 	_, err = signedKeyFile.WriteString(signedKey)
 	return err
 }
+
+// FindUserPubKeys checks every pubkey in `keys` list and returns only existsing keys (or error if
+// all pubkeys don't exist)
+func FindUserPubKeys(keys []string) ([]string, error) {
+	var found = []string{}
+	for _, key := range keys {
+		pubKeyPath, err := UserPubKeyExists(key)
+		if err != nil {
+			return nil, err
+		}
+		if pubKeyPath != "" {
+			found = append(found, pubKeyPath)
+		}
+	}
+
+	if len(found) == 0 {
+		var errStr = fmt.Sprintf(`user SSH keys at %s doesn't exist.
+
+Please generate at least one with command like this :
+
+`, strings.Join(keys, ", "))
+
+		// keys list is not explicitly set, suggest generating ed25519 as default
+		if reflect.DeepEqual(keys, DefaultSSHKeys) {
+			errStr += "\tssh-keygen -f ~/.ssh/id_ed25519 -t ed25519\n"
+		} else {
+			for _, key := range keys {
+				suggestedSSHKey := strings.Replace(key, ".pub", "", 1)
+				errStr += fmt.Sprintf("\tssh-keygen -f %s -t %s\n", suggestedSSHKey, chooseSSHKeyType(suggestedSSHKey))
+			}
+		}
+
+		return nil, fmt.Errorf(errStr)
+	}
+	return found, nil
+}
+
+func chooseSSHKeyType(key string) string {
+	switch {
+	case strings.Contains(key, "dsa"):
+		return "dsa"
+	case strings.Contains(key, "ecdsa_sk"):
+		return "ecdsa-sk"
+	case strings.Contains(key, "ecdsa"):
+		return "ecdsa"
+	case strings.Contains(key, "ed25519_sk"):
+		return "ecdsa-sk"
+	case strings.Contains(key, "rsa"):
+		return "rsa"
+	default:
+		return "ed25519"
+	}
+}
```

Comment:
The output doesn't seem ok for files with "ecdsa" or "ed25519_sk"
```
# go run main.go -k \~/.ssh/id_ecdsa.pub,~/.ssh/id_dsa.pub,~/.ssh/id_ecdsa_sk.pub,~/.ssh/id_ed25519.pub,~/.ssh/id_ed25519_sk.pub,~/.ssh/id_rsa.pub
Error: user SSH keys at ~/.ssh/id_ecdsa.pub, ~/.ssh/id_dsa.pub, ~/.ssh/id_ecdsa_sk.pub, ~/.ssh/id_ed25519.pub, ~/.ssh/id_ed25519_sk.pub, ~/.ssh/id_rsa.pub doesn't exist.

Please generate at least one with command like this :

        ssh-keygen -f ~/.ssh/id_ecdsa -t dsa
        ssh-keygen -f ~/.ssh/id_dsa -t dsa
        ssh-keygen -f ~/.ssh/id_ecdsa_sk -t dsa
        ssh-keygen -f ~/.ssh/id_ed25519 -t ed25519
        ssh-keygen -f ~/.ssh/id_ed25519_sk -t ecdsa-sk
        ssh-keygen -f ~/.ssh/id_rsa -t rsa
exit status 1
```

Reply:
I should have written test for this function before I made PR, thank you! Test added, function fixed